grails-app/services/org/broadinstitute/orsp/PermissionService.groovy

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.orsp
+
+import grails.gorm.transactions.Transactional
+import groovy.util.logging.Slf4j
+
+@Transactional
+@Slf4j
+class PermissionService implements UserInfo{
+
+    UserService userService
+
+    // get issue's collaborators as a List<String>
+    def getIssueCollaborators(issue) {
+        def collaborators = issue.extraProperties.findAll ({ it.name == 'collaborator' }).collect { property -> property.value }
+        collaborators
```

Comment:
This can be a one-liner for simplicity: 
```suggestion
List<String> getIssueCollaborators(issue) {
        issue.extraProperties.findAll ({ it.name == 'collaborator' }).collect { property -> property.value }
}
```
Here and throughout.